SignalServiceKit/src/Messages/OWSDisappearingMessagesJob.m

```diff
@@ -85,7 +85,10 @@ + (dispatch_queue_t)serialQueue
     static dispatch_queue_t queue = nil;
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
-        queue = dispatch_queue_create("org.whispersystems.disappearing.messages", DISPATCH_QUEUE_SERIAL);
+        NS_VALID_UNTIL_END_OF_SCOPE NSString *label = [OWSDispatch createLabel:@"disappearingMessages"];
```

Comment:
Newbie question: why did this change from `disappearing.messages` to `disappearingMessages`?

Reply:
I figured this was a typo from before but I can change it back if you'd prefer. Like I mentioned above, queue labels are really only used for debugging. I think it'd make sense to have it called `org.whispersystems.signal.disappearing.messages` if there were a family of `org.whispersystems.signal.disappearing.*` queues to go along with it.

The whole subsystem that handles disappearing messages is called just that: "Disappearing Messages". There's no separate "Disappearing Photos" or whatever. So to me, it makes sense for the queue to be namespaced as if it's a singular concept. But if you'd prefer the other format I'm happy to undo it!